### DISCUSSION TITLE
LibreOffice window is floating

### DISCUSSION THREAD

**Discussion by mattharrison** [nikitabobko/AeroSpace#740] [questions-and-answers]
Title says it all. LibreOffice documents float. I would like them to tile.

Happy to send any info that is useful.

**Comment by wm-r** [comment]
I had the same issue. 
To solve it I got in service mode (`alt-shift-semicolon`) and then pressed `f`. It allows to toggle between floating and tiling layout.

I hope it helps.

**Comment by mattharrison** [comment]
Service mode toggling doesn't work on my machine. Libre office floats regardless.

**Comment by mattharrison** [comment]
I added this binding
```
ctrl-alt-shift-cmd-f = 'layout tiling floating'
```

Now I can toggle between floating and tiling with hyper-f
